ci/kind/kind-setup.sh

```diff
@@ -23,6 +23,7 @@ ANTREA_IMAGE="projects.registry.vmware.com/antrea/antrea-ubuntu:latest"
 IMAGES=$ANTREA_IMAGE
 ANTREA_CNI=false
 POD_CIDR="10.10.0.0/16"
+SERVICE_CLUSTERIP_CIDR="10.96.0.0/16"
```

Comment:
In https://github.com/antrea-io/antrea/pull/3879, I'm removing POD_CIDR's default value from this script to make it easier to deploy ipv6 and dual-stack cluster, otherwise this parameter must be provided in with correct values. Maybe you should let service cidr defaults to empty as well.

Reply:
Done